### PR TITLE
Handle secure conversation push type

### PR DESCRIPTION
**Pull request by AndriiHorishniiMOC**
Open chat transcript screen from a secure conversation push notification.

[MOB-2130](https://glia.atlassian.net/browse/MOB-2130)

[MOB-2130]: https://glia.atlassian.net/browse/MOB-2130?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by gugalo — APPROVED** [review]

**Review by Pelkar — APPROVED** [review]
LGTM 👍

**Review by DavDo — APPROVED** [review]
LGTM

**Review by andrews-moc — APPROVED** [review]

**Comment by AndriiHorishniiMOC** [comment]
I made additional changes in the testing app and I want you to approve them.
Now when the `QUEUED_MESSAGE` push notification opens and the SDK is not authenticated it will ask for the auth token.

File: widgetssdk/src/main/java/com/glia/widgets/chat/ChatActivity.java [hunk 1]
```diff
@@ -41,6 +41,26 @@ public static Intent getIntent(
                 .putExtra(GliaWidgets.QUEUE_ID, queueId);
     }
 
+    /**
+     * Creates and fills out Intent for starting ChatActivity
+     * @param context - Context object
+     * @param contextId - Context asset ID
```
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/chat/ChatActivity.java:47]: Minor suggestion - we can add "Visitor context asset id" to be more clear about this variable and distinguish it from Android Context.

Comment (andrews-moc) [reply widgetssdk/src/main/java/com/glia/widgets/chat/ChatActivity.java:47]: If you agree with this suggestion, let's also edit it for the `getIntent()` function above.

Comment (andrews-moc) [reply widgetssdk/src/main/java/com/glia/widgets/chat/ChatActivity.java:47]: Agreed to leave it as is now.

**Comment by AndriiHorishniiMOC** [comment]
!squash

**Comment by AndriiHorishniiMOC** [comment]
!merge

**Pull request merged**